### PR TITLE
Custom Entity Models manifest

### PR DESCRIPTION
**Pull request by TacoMonkey11**
Just want to keep to the "One manifest per PR" rule

### CODE REVIEW

**Comment by TacoMonkey11** [comment]
WELL CRAP

**Comment by TacoMonkey11** [comment]
guess who forgot to make a new branch for the next manifest!

**Comment by thefirethirteen** [comment]
Oh dang

File: manifests/D/dorianpb/cem/main.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=../../../../doc/validation-schemas/4.0/main-manifest.json
+
+manifestSpecVersion: "4.0"
+publisher: Digifox03
```
Comment (thefirethirteen) [inline manifests/D/dorianpb/cem/main.yaml]: Wrong publisher!

File: manifests/E/emmaffle/quilt-loading-screen/1.x/1.3.x/1.3.x.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=../../../../../../doc/validation-schemas/4.0/version-manifest.json
```
Comment (thefirethirteen) [inline manifests/E/emmaffle/quilt-loading-screen/1.x/1.3.x/1.3.x.yaml]: Commenting here because I can't comment on the name

Wrong filename!

Comment (thefirethirteen) [reply manifests/E/emmaffle/quilt-loading-screen/1.x/1.3.x/1.3.x.yaml]: I think it should be `1.3.0.yaml`

File: lookup-table.yaml [hunk 3]
```diff
@@ -177,6 +190,17 @@
...
         - fabric
+- id: quilt-loading-screen
+  alternativeNames: ~
+  tags:
+    - cosmetic
+    - decoration
+    - miscellaneous
+    - utility
+  packages:
+    - packageId: Emmaffle.quilt-loading-screen
```
Comment (thefirethirteen) [inline lookup-table.yaml:201]: Either this packageId is wrong, or the folder `/manifests/E/emaffle` you made is wrong

**Review by thefirethirteen — CHANGES_REQUESTED** [review]
Looks good, but needs a little work.

File: lookup-table.yaml (hunk 3 above)
Comment (thefirethirteen) [reply lookup-table.yaml:201]: The folder still seems to be `emaffle`, and this is still `Emaffle`. Which one is it?

**Review by thefirethirteen — CHANGES_REQUESTED** [review]
One last thing...

File: lookup-table.yaml (hunk 3 above)
Comment (thefirethirteen) [reply lookup-table.yaml:201]: The folder is `Emmafle`(one `f`)  while this is `Emmaffle` (double `f`). Which one is it?

**Review by thefirethirteen — CHANGES_REQUESTED** [review]

**Review by thefirethirteen — APPROVED** [review]
Everything looks good now

**Pull request merged**